Xamarin.Forms.Core/WebView.cs

```diff
@@ -39,12 +39,22 @@ public WebView()
 			_platformConfigurationRegistry = new Lazy<PlatformConfigurationRegistry<WebView>>(() => new PlatformConfigurationRegistry<WebView>(this));
 		}
 
+		bool IWebViewController.CanGoBack {
+			get { return CanGoBack; }
+			set { CanGoBack = value; }
+		}
+
 		public bool CanGoBack
 		{
 			get { return (bool)GetValue(CanGoBackProperty); }
 			internal set { SetValue(CanGoBackPropertyKey, value); }
```

Comment:
well, at this point, remove the `internal` on setters, and you can drop the interface as WebView is already public